backend/src/tasks/wappalyzer.ts

```diff
@@ -1,34 +1,85 @@
-import axios from 'axios';
+import axios, { AxiosError } from 'axios';
 import { CommandOptions } from './ecs-client';
 import { getLiveWebsites, LiveDomain } from './helpers/getLiveWebsites';
 import PQueue from 'p-queue';
 import { wappalyzer } from './helpers/simple-wappalyzer';
 
 const wappalyze = async (domain: LiveDomain): Promise<void> => {
   try {
+    console.log(`Executing wapplyzer on ${domain.url}`);
+
     const { data, status, headers } = await axios.get(domain.url, {
+      timeout: 10000,
       validateStatus: () => true
     });
-    const result = wappalyzer({ url: domain.url, data, headers });
-    if (result.length > 0) {
-      domain.service.wappalyzerResults = result.filter((e) => e?.technology);
-      await domain.service.save();
+
+    // only respose with HTTPStatusCodes lesser than 4xx are wappalizyed
+    if (status < 400) {
```

Comment:
I don't think we want to only limit to responses less than 400. This is because the headers can still show useful information even if there is a 400 response -- for example, the headers can still indicate technologies such as Apache that are used, and are detected by wappalyzer.

Reply:
Done